### PR TITLE
Update to Julia v1.6

### PR DESCRIPTION
**Pull request by lmh91**
PR to prepare for Julia v1.6. 
Do not merge till v1.6 is released.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -20,7 +20,7 @@ jobs:
       matrix:
         version:
           - '1.0'
-          - '1.5'
+          - '1.6'
```
Comment (oschulz) [inline .github/workflows/ci.yml:23]: We should set this to '1', that will automatically use the latest Julia release.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -70,7 +70,7 @@ jobs:
       - uses: actions/checkout@v2
       - uses: julia-actions/setup-julia@latest
         with:
-          version: '1.5'
+          version: '1.6'
```
Comment (oschulz) [inline .github/workflows/ci.yml:73]: We should set this to '1', that will automatically use the latest Julia release.

**Review by oschulz — CHANGES_REQUESTED** [review]

**Comment by oschulz** [comment]
Nice - like this, we don't have to wait for the v1.6 release to merge it.

**Comment by lmh91** [comment]
That's true. 

Should we actually drop v1.0 already @oschulz?
v1.6 should become the next LTS version, right? 
Or do we wait till v1.7?

**Comment by oschulz** [comment]
Let's set Julia v1.3 as the minimum right now (and also test on v1.3), many package have done that. If v1.6 seems likely to become the new LTS, but this will only be confirmed during the v1.7 release process, from what I understand.

**Comment by lmh91** [comment]
Okay, than I will leave v1.0 inside for now.

**Pull request merged**